continuous_delivery/deploy_and_basics.rst

```diff
@@ -23,17 +23,7 @@ Deliver (Simple)
       #. … a test system with a custom Git branch or tag
       #. … a production system with a Docker image previously installed on the test system
 
-   .. todo:: link "Docker image" to high level overview
-
-.. figure:: deploy_and_basics_static/tc_run_menu.png
-   :name: run menu
-
-   *Run* menu
-
-#. Click on **Run** in the `main page`_
-#. (optional) adjust the `dump mode`_ in the `run menu <#run-menu>`__
-#. Click **Run Build** in the `run menu <#run-menu>`__
-
+Just click on **Run** in the `main page`_ to start a deployment.
```

Comment:
You need to click run in the main menu and the in the opened window again, right?

Reply:
No, the window isn't opened anymore. If you click run the build will start with the default parameters.

Reply:
Can you make sure the window is opened again? The risk of anyone deploying an installation is too high otherwise.